File: Raster.Samples/prime.py
import time

def isPrime(num):
    count = 0;
    for i in range(1, num+1):
        if num % i == 0:
            count += 1
    return count == 2;

def findPrimes(start, stop):
    count = 0;
    for x in range(start, stop+1):
        if isPrime(x):
            count += 1
    print("%d primes are in this range" % count)

start = int(round(time.time() * 1000))

findPrimes(0, 20000)

time = int(round(time.time() * 1000)) - start

print('%dms time to find primes' % time)

File: Raster.Samples/prime.js

const console = require('console');

const isPrime = (num) => {
    let timesReminderEqualsZero = 0;
    for (let i = 1; i <= num; ++i) {
        if (num % i == 0) {
            ++timesReminderEqualsZero;
        }
    }
    return timesReminderEqualsZero == 2;
}

const findPrimes = (start, stop) => {
    let count = 0;
    for (let i = start; i <= stop; ++i) {
        if (isPrime(i)) {
            ++count;
        }
    }

    console.log(count + " primes are in this range");
}

const startTime = (new Date()).getTime();
findPrimes(0, 20000);
console.log(((new Date()).getTime() - startTime) + 'ms time to find primes');
